lib/create-space-api.ts

```diff
@@ -27,7 +27,8 @@ export type ContentfulSpaceAPI = ReturnType<typeof createSpaceApi>
  * @prop {object} entities - Object with wrapper methods for each kind of entity
  * @return {ContentfulSpaceAPI}
  */
-export default function createSpaceApi({ http }: { http: AxiosInstance }) {
+export default function createSpaceApi({ adapter }: { adapter: Adapter }) {
+  const http = (adapter as RestAdapter).http
```

Comment:
Why is this here? Shouldn't we get rid of `http` and only use `makeRequest`? Is it maybe temporary?

Reply:
This is just to make sure the whole thing still compiles. This will be eventually removed.